### ISSUE TITLE
Bad ratio with github version (not in the package version)

### ISSUE THREAD

**Issue by demonixis** [artoolkit/arunity5#172]
Hi,

I have a problem with the latest sources of this repo because the ratio is bad. Please take a look at these to screenshots

![image](https://user-images.githubusercontent.com/1017126/27533155-d9c908e0-5a62-11e7-961b-1b65404fba6a.png)

And

![image](https://user-images.githubusercontent.com/1017126/27533171-eb9d0742-5a62-11e7-8498-faded09f8478.png)

The package provided on the website is good, I just patched it with orientation support. But it's impossible to publish an app on the Android Store because of an OpenSSL problem.  Updating the sources from github have broken my project.

I tried to change the scale of my scene, it's very hacky and doesn't work well on mobile. Please can you try it ? If you don't have the time, please tell my howto fix that. 

Thank you.

**Comment by ThorstenBux** [comment]
Here is more background about SSL. https://github.com/artoolkit/artoolkit5/issues/215

Maybe you can follow this and use the download version. If you are having trouble with the latest from gitHub. Otherwise hard to help because if I understand correctly you "I just patched it with orientation support" which might cause this ratio issues.

**Comment by demonixis** [comment]
Hi,

Thanks for the quick answer, I'll try to follow your link. To be clear, the first version of the app is an Android App which requires screen orientation support. For that I've replaced the code of `ARController` by the one provided in #91 and it worked great.

Now when I updated my sources and libs using this repository, my problems started. Obviously my modification of `ARController` was replaced by the version of this repository. So without changing anything, I started the application on my PC and discovered this strange ratio error. I tried on Android too, but I've this error too. Updating `ARController` with the code of #91 only bring screen orientation support, nothing more or less.

Also I noticed that the `ARCamera` uses a the `DontClear` flag which is bad sometime because the mesh is not displayed. It can be easily fixed using the `Depth Only` flag (I had this problem with the download version as well).

I hope these informations can help.

Regards,
